fhir-server-test/src/test/java/com/ibm/fhir/server/test/FHIRPatchTest.java

```diff
@@ -540,7 +551,103 @@ public void testFhirPatchMoveOperation() throws Exception {
 
         Assert.assertEquals(updatedPatient, responsePatient);
     }
+    
+    @Test(groups = { "fhir-patch" })
+    public void testBundlePatch() throws Exception {
+        
+        WebTarget target = getWebTarget();
+        // Build a new Patient and then call the 'create' API.
+        Patient patient = buildPatient();
+
+        Entity<Patient> entity = Entity.entity(patient, FHIRMediaType.APPLICATION_FHIR_JSON);
+        Response response = target.path("Patient/" + patient.getId()).request().put(entity, Response.class);
+        assertResponse(response, Response.Status.CREATED.getStatusCode());
+               
+        // Get the patient's logical id value.
+        String patientId = getLocationLogicalId(response);
 
+        //replace patch request for the patient
+        Parameters replacePatch = Parameters.builder()
+                .parameter(Parameter.builder()
+                    .name(string("operation"))
+                    .part(Parameter.builder()
+                        .name(string("type"))
+                        .value(Code.of("replace"))
+                        .build())
+                    .part(Parameter.builder()
+                        .name(string("path"))
+                        .value(string("Patient.active"))
+                        .build())
+                    .part(Parameter.builder()
+                        .name(string("value"))
+                        .value(com.ibm.fhir.model.type.Boolean.FALSE)
+                        .build())
+                    .build())
+                .build();
+        
+        //Delete Patch request for the patient
+        Parameters deletePatch = Parameters.builder()
+                .parameter(Parameter.builder()
+                    .name(string("operation"))
+                    .part(Parameter.builder()
+                        .name(string("type"))
+                        .value(Code.of("delete"))
+                        .build())
+                    .part(Parameter.builder()
+                        .name(string("path"))
+                        .value(string("Patient.active"))
+                        .build())
+                    .build())
+                .build();
+        
+        //Creating bundle containing multiple patch request for the patient
+        Bundle.Builder patchBundleBuilder=Bundle.builder();
+        String patientUrl=new StringBuilder(target.getUri().toString()).append("Patient/").append(patientId).toString();
```

Comment:
Based on https://www.hl7.org/fhir/bundle-definitions.html#Bundle.entry.request.url I would expect this to be a relative URI, not an absolute URI.
```suggestion
        String patientUrl = "Patient/" + patientId;
```